POLMED/src/components/button/index.tsx

```diff
@@ -0,0 +1,10 @@
+type Props = {
+    content: string;
+    className: string;
```

Comment:
Możesz w TypeScripcie z góry zakładać jaki będziesz mieć string, wyeliminuje to literówki.
![image](https://user-images.githubusercontent.com/33033063/205491238-3d381a64-9c15-4940-8b74-34a341ad49fd.png)